### ISSUE TITLE
Sending DELETE request without id causing double slashes url path

### ISSUE THREAD

**Issue by gaghan430** [mahdaen/metal-data#2]
so when we use something like `collection.delete(null, { suffix: 'delete' })`, 
we expect to get `/endpoint/delete`, but what we get is `/endpoint//delete`

I guess we need to filter out falsy values from this segments https://github.com/mahdaen/metal-data/blob/master/src/request.ts#L84

**Comment by mahdaen** [comment]
@gaghan430 Good point to prevent falsy values, will work on that! But for such operation I suggest to use the `.request()` method instead, to keep the coding style clean.
